index.html

```diff
@@ -18,12 +18,113 @@
 
     <div class="container">
       <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
-          </div>
-      </div>
+          <div class="col-lg-12">
+            <ul class="nav justify-content-center">
+              <li class="nav-item">
+                <a class="nav-link active" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About Me</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">References</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Contact Me!</a>
+              </li>
+            </ul>
+          </div><!--NavBar-->
+      <div class="row1">
+          <div class="jumbotron">
+            <h1 class="display-4">Hello, world!</h1>
+            <img class="pix" src="img/">
```

Comment:
you may notice image link is broken on site.  You will need to add an image inside your img folder and put the path to it here.